packages/grafana-data/src/types/app.ts

```diff
@@ -56,6 +57,7 @@ export interface AppPluginMeta<T extends KeyValue = KeyValue> extends PluginMeta
 }
 
 export class AppPlugin<T extends KeyValue = KeyValue> extends GrafanaPlugin<AppPluginMeta<T>> {
+  private _exportedComponentConfigs: PluginExportedComponentConfig[] = [];
```

Comment:
Throughout the codebase, this is now being referred to as exportedComponentsConfigs. Not sure about the `exported` prefix though. Maybe makes more sense to call it `exposed`?